cli/src/benchmark/mod.rs

```diff
@@ -213,12 +215,13 @@ impl BenchmarkCommands {
 
 					output.push(result);
 				}
-				client.client_api.close().unwrap();
 
-				let balance = get_balance(cli, trusted_args, &client.account.public().to_string());
+				let balance = get_balance(client.account, shard, &client.client_api);
 				println!("Balance: {}", balance.unwrap_or_default());
 				assert_eq!(client.current_balance, balance.unwrap());
 
+				client.client_api.close().unwrap();
+
```

Comment:
Instead of adding additional getter executions to measure timings, I refactored the existing `get_balance` getters and added a timing measurement

Reply:
Looks good 👍